addOns/pscanrulesBeta/CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+## [20] - 2019-07-06
```

Comment:
This should be removed, the changes should be added to the Unreleased section.

Reply:
I assume you mean by moving it above the ## Unreleased marker?

Reply:
No he means don’t add 20 and the date, but your comment/addition in unreleased

Reply:
OK. We'll fix, but if you could be a bit more specific in your comments, we wouldn't have to go back and forth on these, just like the next comment where we didn't understand that one either.

Reply:
Sorry, I thought it was clear ("this" is the referenced line and "changes" whatever was changed in the add-on), also, I'm not saying more here (or much differently) than what is said in the [CONBTRIBUTING](https://github.com/zaproxy/zap-extensions/blob/master/CONTRIBUTING.md) file (if that's not clear either feel free to suggest or PR changes).
(Note that the changes are still not in the right place, they should be in the Unreleased section not before/above.)

For the other comment I thought the "name of the scanner" would hint to its `getName()` method (and end up in the `Messages.properties` file).